pdf/src/campPrint/scheduleEntry/ScheduleEntry.vue

```diff
@@ -16,22 +16,21 @@
       <Text class="schedule-entry-date">{{ startAt }} - {{ endAt }}</Text>
     </View>
     <View v-if="showHeader" class="schedule-entry-header">
-      <View class="schedule-entry-header-metadata" style="padding-right: 7pt">
-        <Text>
-          <Text v-if="activity.location" class="schedule-entry-header-metadata-label">
-            {{ $tc('entity.activity.fields.location') }}:
-          </Text>
-          {{ activity.location }}
-        </Text>
+      <View class="schedule-entry-header-metadata">
+        <View class="schedule-entry-header-metadata-entry"
+          ><Text v-if="activity.location" class="schedule-entry-header-metadata-label"
+            >{{ $tc('entity.activity.fields.location') }}:</Text
+          ><Text>{{ activity.location }}</Text></View
+        >
       </View>
+      <View class="schedule-entry-header-divider" />
       <View class="schedule-entry-header-metadata">
         <View class="schedule-entry-header-metadata-entry">
           <Text
             v-if="activity.activityResponsibles().items.length"
             class="schedule-entry-header-metadata-label"
+            >{{ $tc('entity.activity.fields.responsible') }}:</Text
```

Comment:
unnecessary formatting change.
And i think it was better before.

Reply:
Nope, same as https://github.com/ecamp/ecamp3/pull/4071#discussion_r1412811230